api-test/src/main/resources/pms/OIDCClientEnhancement/GetOIDCClientByClientId/GetOIDCClientByClientIdNegTC.yml

```
@@ -0,0 +1,91 @@
+GetOIDCClientByClientIdNegTCV2:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)